### PR TITLE
Fix more breakpoint handling on aarch64

### PR DESCRIPTION
**Pull request by yuyichao**
Fix code that assumes the pc points to after the breakpoint instruction
after hitting the breakpoint.

### CODE REVIEW

File: src/remote_code_ptr.h [hunk 1]
```diff
@@ -60,6 +60,19 @@ class remote_code_ptr {
         return *this;
     }
   }
+  remote_code_ptr continue_executed_bkpt(SupportedArch arch) {
```
Comment (khuey) [inline src/remote_code_ptr.h:63]: This I think needs a better name. I have no idea what it's supposed to do.

Comment (yuyichao) [reply src/remote_code_ptr.h:63]: I'm very bad at coming up with names so I'm very open to suggestions....

I was initially going to call it "redo_****" but that feels really like making it redo the breakpoint, which is exactly the opposite.... so...

Comment (Keno) [reply src/remote_code_ptr.h:63]: advance_past_executed_bkpt?

Comment (khuey) [reply src/remote_code_ptr.h:63]: that sounds good to me.

Also put the aarch64 case above default ...

Comment (yuyichao) [reply src/remote_code_ptr.h:63]: Sure,

> Also put the aarch64 case above default ...

What about the undo function? I've got confused by that order before and I just copied it here.....

Comment (khuey) [reply src/remote_code_ptr.h:63]: Yeah you can fix that too.

Comment (yuyichao) [reply src/remote_code_ptr.h]: Done. It seems that the ordering could be for the `RR_FALLTHROUGH`. I changed that to a unreachable instead as already used elsewhere.

**Pull request merged**